build/pkgs/meson_python/spkg-configure.m4

```diff
@@ -1,3 +1,3 @@
 SAGE_SPKG_CONFIGURE([meson_python], [
-  SAGE_PYTHON_PACKAGE_CHECK([meson-python])
+  SAGE_PYTHON_PACKAGE_CHECK([meson_python])
```

Comment:
Did this cause a problem? I recall making the inverse change but not the reason why. The argument to `SAGE_PYTHON_PACKAGE_CHECK` should be the name of the python package (not the sage package), although both names seem to work when passed to `pkg_resources.require()`. In that regard `meson-python` more closely matches what's in e.g. `install-requires.txt`

Reply:
dashes don't work as part of environment variable names - see https://groups.google.com/g/sage-release/c/1wOBmhvNJqc/m/8IDtj6Y4BQAJ

Reply:
on the Python side, dashes and underscores in distribution names are interchangeable and get normalized as needed

Reply:
Ok, it still works for me with the underscore FWIW.